build.yaml

```diff
@@ -347,6 +349,7 @@ common_cli:
     # bootstrapping. These are copied from the Jenkins Master /etc/slave_config
     # directory.
     export FLOCKER_FUNCTIONAL_TEST_CLOUD_CONFIG_FILE=/tmp/acceptance.yaml
+    export FLOCKER_FUNCTIONAL_TEST_CLOUD_CONFIG_SECTION=storage-drivers.gce
     export FLOCKER_FUNCTIONAL_TEST_CLOUD_PROVIDER=gce
```

Comment:
Is this necessary still? Prefer duplicate data over the wrong abstraction, but this looks like it might actually just be the wrong abstraction.

If it's not easy to remove for other structural reasons, feel free to ignore.